docs/toctree.rst

```diff
@@ -0,0 +1,21 @@
+Overview
+========
+
+.. toctree::
+
+   index
+   start
+   usage/intro
+   usage/mocking
+   usage/spying
+   usage/call_count
+   usage/argument_matching
+   advanced/builtin_functions
+   advanced/call_order
+   advanced/chained_methods
+   advanced/generators
+   advanced/multiple_return
+   advanced/new_instances
+   advanced/replacing_functions
+   advanced/state
+   compare
```

Comment:
Is there no way to generate this ? I'm afraid it's quickly going to be outdated :)

Reply:
Yeah, I don't like having duplicate table of contents here either. I'll look into it. Maybe we could test in CI that Sphinx build doesn't raise any warnings or errors.

Reply:
I added CI job to test Sphinx documentation generation:
https://github.com/flexmock/flexmock/pull/105/commits/8b39f661ac0b1bed36aca1cfc098965d72466ccb